DAC/src/tlv_dac.c

```diff
@@ -91,19 +92,35 @@ static void _tlv_dac_initI2cPins()
 	I2C_StructInit(&i2cInterfaceSettings);
 }
 
-/**
+/*
```

Comment:
Extra star should be used on headers for functions. The change should be reverted.